logic/raid_list.php

```diff
@@ -35,9 +39,12 @@ function raid_list($update)
                     LEFT JOIN   gyms
                     ON          raids.gym_id = gyms.id
                     LEFT JOIN   users
-                    ON          raids.user_id = users.user_id
-		      WHERE     raids.id = {$iqq}
-                      AND       end_time>UTC_TIMESTAMP()
+		    ON          raids.user_id = users.user_id
+		     WHERE     ( raids.id = {$iqq} or gyms.gym_name like '%{$db->real_escape_string($update['inline_query']['query'])}%' )
```

Comment:
With PDO real_escape_string is gone and you'll need to change all of this into a prepared statement anyway. LIKE wildcards need a bit of special handling, this one has some good examples, especially the second answer: https://stackoverflow.com/questions/583336/how-do-i-create-a-pdo-parameterized-query-with-a-like-statement

There's examples of prepared statements now all over the codebase, here's a simple sample, your LIKE wildcards will need the finesse discussed above.
```
       $sql = "INSERT INTO   gyms
            SET       lat = :lat,
                          lon = :lon,
                          gym_name = :name,
                          address = :address";
        $stmt = $dbh->prepare($sql); 
        $stmt->execute([
          'lat' => $lat,
          'lon' => $lon,
          'name' => $name,
          'address' => $address
        ]);
```